### PR TITLE
add install to help

### PR DESCRIPTION
**Pull request by takkanm**
There was no "install" to help message.

### CODE REVIEW

File: src/bin/cargo.rs [hunk 1]
```diff
@@ -49,8 +49,11 @@ Some common cargo commands are:
     run         Build and execute src/main.rs
     test        Run the tests
     bench       Run the benchmarks
+
+
     update      Update dependencies listed in Cargo.lock
     search      Search registry for crates
+    install     Install a Rust binaly
```
Comment (killercup) [inline src/bin/cargo.rs]: s/binaly/binary

Comment (takkanm) [inline src/bin/cargo.rs]: oh :scream: I fixed.

File: src/bin/cargo.rs [hunk 2]
```diff
@@ -49,8 +49,11 @@ Some common cargo commands are:
     run         Build and execute src/main.rs
     test        Run the tests
     bench       Run the benchmarks
+
+
```
Comment (alexcrichton) [inline src/bin/cargo.rs]: Were these new lines intentional?

Comment (takkanm) [inline src/bin/cargo.rs]: sorry. I remove these line and squash commits.

**Comment by alexcrichton** [comment]
@bors: r+ 5a427b47f7e13af7962b059c4aed7b484089c8ec

Thanks!

**Pull request merged**